include/gmsh.tcc

```diff
@@ -227,19 +239,26 @@ void GMSH<Tdim, Tvertices>::compute_material_points() {
 template <unsigned Tdim, unsigned Tvertices>
 void GMSH<Tdim, Tvertices>::compute_stresses() {
```

Comment:
Should compute_stress be part of mesh?

Reply:
I don't think so... so where to put? I personally recommend to move it to another header file if possible